frontend/src/actions/rent/index.ts

```diff
@@ -0,0 +1,48 @@
+import { Dispatch } from 'redux';
+import * as api from '../../api';
+import {
+    CREATE_RENTAL_REQUEST,
+    CREATE_RENTAL_REQUEST_ERROR,
+    RENTAL_REQUEST_LOADING,
+} from '../types';
+import { getErrors } from '../error';
+
+export const createRentalRequest =
+    (requestData?: any) => async (dispatch: Dispatch<any>) => {
+        dispatch({ type: RENTAL_REQUEST_LOADING });
+        try {
+            const formData = formatData(requestData);
+            const { data } = await api.createRentalRequest(formData);
+            dispatch({ type: CREATE_RENTAL_REQUEST, data: data });
```

Comment:
lil trick: cuando se llama igual, no es necesario el 2 puntos:
```suggestion
            dispatch({ type: CREATE_RENTAL_REQUEST, data });
```

Es lo mismo igual, como gustes

Reply:
I knew it but I manquiated it, gonna fix later today

Reply:
si en realidad no es un error, si queres dejarlo asi es lo mismo